.github/workflows/reusable-image-scan-and-release-ghcr.yml

```diff
@@ -1,4 +1,6 @@
 name: Scan and Release Docker image to GHCR
+permissions:
+  contents: read
```

Comment:
Either add `packages: write` for the entire workflow or for the `release-tag` and `release-main` steps as this pipeline pushes images to github container registry